modules/ROOT/pages/creating-a-skin.adoc

```diff
@@ -90,30 +113,60 @@ Your skin.less file should now look like this:
 
 @background-color: red;
 ----
++
+. Save the file and check the changes in a browser. Select the skin from the _Skin menu_.
+. The editor should now display with a red background.
+
+image::SDK-for-silver-custom-example.png[{productname} Skin SDK for the Silver theme]
 
-Switch to the web browser. Select your skin from the _Skin menu_. It should show a fiery red editor
+This method allows customization of {productname} by modifying predefined variables, such as toolbar spacing or letter spacing.
 
-image::SDK-for-silver-custom-example.png[{productname} skin SDK for Silver theme]
+[TIP]
+Adjust {productname} configurations in `+modules/oxide/src/demo/index.html+` to match specific use cases.
 
-This is how you skin {productname}: copy variables from the files in the theme folder and paste them into your skin file. There are variables for most things, like spacing between toolbar buttons to letter spacing. Simple yet powerful.
+== Creating a Content CSS File
 
-TIP: You can change the {productname} config in `+modules/oxide/src/demo/index.html+` to suit your particular use case.
+To modify the appearance of content inside the editor (e.g., headings, lists, quotes), create a content CSS file:
 
-== Creating a content CSS file
+. Create a folder in `+modules/oxide/src/less/skins/content/+` and add a `content.less` file. Alternatively, duplicate an existing content CSS file.
+. Rebuild the project to recognize the newly created content skin.
++
+[source,sh]
+----
+yarn build
+----
+. Start the development server:
++
+[source,sh]
+----
+yarn oxide-start
+----
++
+[TIP]
+If the server is already running, restart it (`+Ctrl+C+`) and run the command again.
 
-To update the appearance of the content within the editor, such as headings, quotes, lists, etc... you create a content css. These are located in `+modules/oxide/src/less/skin/content/+`
+. Add styles for relevant elements (`h1` to `h6`, `a`, `blockquote`, `code`, `table`, etc.). For example:
++
+[source,less]
+----
+h1 {
+  color: red;
```

Comment:
I wonder why we don't use variables in contents... looks ike we don't import `theme` folder

Reply:
Agree, this might be something we can improve later.